subcommands/targets/add.go

```diff
@@ -26,10 +26,10 @@ type Targets map[string]*client.Target
 func init() {
 	addCmd := &cobra.Command{
 		Use:   "add",
-		Short: "Compose and add Targets to Factory's  TUF targets metadata",
+		Short: "Compose and add Targets to Factory's TUF Targets metadata",
 		Run:   doAdd,
 		Long: `
-Compose new Targets out of the latest Targets tagged with the specified source tag and the specified via the command arguments either OSTree commit hashes or App URIs.
+Compose new Targets out of the latest ones with the specified source tag via the command arguments—either OSTree commit hashes or App URIs.
```

Comment:
Don't we need a space on both sides of a dash?
```suggestion
Compose new Targets out of the latest ones with the specified source tag via the command arguments — either OSTree commit hashes or App URIs.
```

Reply:
No. Technically no spaces, however it is not a hard rule either way.

Reply:
I've seen you use spaces in some places, and no spaces in others.

So, we need at least consistency.
If I have a voting right, I vote to add spaces. That looks more cross-lingual / cross-culture.